src/main/navigation/navigation.c

```diff
@@ -100,18 +100,18 @@ PG_RESET_TEMPLATE(navConfig_t, navConfig,
 
         // General navigation parameters
         .pos_failure_timeout = 5,               // 5 sec
-        .waypoint_radius = 100,                 // 2m diameter
-        .waypoint_safe_distance = 10000,        // centimeters - first waypoint should be closer than this
-        .max_auto_speed = 300,                  // 3 m/s = 10.8 km/h
-        .max_auto_climb_rate = 500,             // 5 m/s
-        .max_manual_speed = 500,
+        .waypoint_radius = 1000,                // 10m diameter
```

Comment:
Many of us fly missions also on quads where `nav_wp_radius=1m` is perfectly reasonable. For planes, you'll also "hit" the waypoint if you bypass it and it's now behind (more than 100 deg turn required compared to original bearing). I'd keep the 1m default.

Reply:
> 
> 
> . For planes, you'll also "hit" the waypoint if you bypass it and it's now behind (more than 100 deg turn required compared to original bearing). 
Sure that works ?
Here is my plane looping around the 8th WP ("+1") forever.Video is cut short but I stopped waiting after about 6 turns :